taxonomies/additives.txt

```diff
@@ -15766,7 +15767,7 @@ bg:E901, Пчелен восък, бял и жълт, Бял восък
 ca:E901, Cera d'abelles
 cs:E901, Včelí vosk, bílý a žlutý, Bílý vosk, E 901
 da:E901, Bivoks, hvidt og gult, Hvidt voks, E-901
-de:E901, Bienenwachs; weiss und gelb, weißes Wachs, Bienenwachs, E 901, Myricin, Cera Flava
+de:E901, Bienenwachs; weiß und gelb, weißes Wachs, Bienenwachs, E 901, Myricin, Cera Flava, Bienenwachs; weiß und gelb
```

Comment:
; vs ,

Reply:
This doesn't work in fact, we cannot have commas in an additive name.

"Beeswax; white and yellow," --> Needs to be changed to "Beeswax, white beeswax, yellow beeswax, white and yellow beeswax, yellow and white beeswax".

Same for all languages.

Otherwise it makes "White and yellow" a synonym of "Beeswax".

Reply:
Thanks, what about brackets? I saw things like this: `beeswax (white and yellow)`. Also sometimes things like `beeswax \, white and yellow`.

Btw, on the page for [`en:e901`](https://world.openfoodfacts.org/ingredient/e901) the link to  [`en:white-and-yellow-beeswax`](https://world.openfoodfacts.org/ingredient/white-and-yellow-beeswax) is unreachable, as it links back to `en:901`, probably because in additives.txt it is synonym to it?

Reply:
same things for brackets, parenthesis etc. : they should not be used in the names and synonyms of additives, it won't work.

Reply:
The E901 link is because there are also entries for E901 / beeswax in the ingredients taxonomy, we will need to remove them from the ingredients taxonomy, and add the extra translations to the additives taxonomy instead. cc @aleene

Reply:
1.) Here it seems to work with the brackets: [https://world.openfoodfacts.org/additive/e285-sodium-tetraborate (borax)](https://world.openfoodfacts.org/additive/e285-sodium-tetraborate%20(borax)), but it seems it needs the proper single words still. And e.g. here it doesn't work so well: https://world.openfoodfacts.org/additive/e342-ammonium-phosphates-i-monoammonium-phosphate-ii-diammonium-phosphate
2.) About additives: is it in principle save to remove them from `ingredients.txt`? (including transferring the translations) if they have a 1:1 partner in the `additives.txt`?